upload + download werkt op dit punt.

ChrisOrdi opened the pull request.

ChrisOrdi committed:
upload + download werkt op dit punt.

pietervoorschot reviewed (commented):
Wat is het verschil in code tussen regel 78 t/m 80 en regel 211 t/m 212?

Wat doet de if/else met switch anders dan de switch case die eronder staat?

pietervoorschot committed:
Added /help function to show all commands.

pietervoorschot committed:
Removed old and redundant code.
Cleaned up the code for readability.
Resolved warnings.
Placed all the logic behind file handling to ClientHandler (new class). This should be split up more later to have less merge conflicts and to make the methods testable.

ChrisOrdi committed:
sync werkt, upload werkt naar behoren, download is niet meer nodig nu in principe, kunnen we laten staan, delete niet gecheckt maar hoort te werken.

ChrisOrdi committed:
sync werkt, upload werkt naar behoren, download is niet meer nodig nu in principe, kunnen we laten staan, delete niet gecheckt maar hoort te werken.

pietervoorschot committed:
Moved the methods to their own classes so everything is testable now.

ChrisOrdi committed:
delete en upload func werken.

ChrisOrdi committed:
added test classes, tests still need to be made.

ChrisOrdi committed:
added tests.

ChrisOrdi committed:
added some func

ChrisOrdi committed:
update van 30 mei 2024

pietervanoorschot committed:
Updated unit tests.
